src/project.rs

```diff
@@ -213,3 +226,35 @@ pub fn collect_source(
     }
     Ok(())
 }
+
+fn check_app_file_version_matches(
+    root: impl AsRef<Path>,
+    project_config: &PackageConfig,
+) -> Result<(), Error> {
+    let mut app_src_path = root.as_ref().to_path_buf();
+    app_src_path.push("src");
+    app_src_path.push(format!("{}.app.src", &project_config.name));
+
+    let contents = std::fs::read_to_string(&app_src_path)
```

Comment:
Could we make this function return `Ok(())` if the .app file isn't present? That will be the case for mix users.

Reply:
Done.